src/main/java/com/exasol/adapter/document/mapping/PropertyToBoolColumnMapping.java

```diff
@@ -36,6 +33,95 @@ public ColumnMapping withNewExasolName(final String newExasolName) {
         return this.toBuilder().exasolColumnName(newExasolName).build();
     }
 
+    /**
+     * Builder for {@link PropertyToBoolColumnMapping}.
+     */
+    public abstract static class PropertyToBoolColumnMappingBuilder<C extends PropertyToBoolColumnMapping, B extends PropertyToBoolColumnMapping.PropertyToBoolColumnMappingBuilder<C, B>>
+            extends AbstractPropertyToColumnMapping.AbstractPropertyToColumnMappingBuilder<C, B> {
+        private ConvertableMappingErrorBehaviour notBooleanBehavior;
+
+        @Override
+        protected B fillValuesFrom(final C instance) {
+            super.fillValuesFrom(instance);
+            PropertyToBoolColumnMapping.PropertyToBoolColumnMappingBuilder.fillValuesFromInstanceIntoBuilder(instance,
+                    this);
+            return self();
+        }
+
+        private static void fillValuesFromInstanceIntoBuilder(final PropertyToBoolColumnMapping instance,
+                final PropertyToBoolColumnMapping.PropertyToBoolColumnMappingBuilder<?, ?> b) {
+            b.notBooleanBehavior(instance.notBooleanBehavior);
+        }
+
+        @Override
+        protected abstract B self();
+
+        @Override
+        public abstract C build();
+
+        /**
+         * @param notBooleanBehavior behaviour in case value is not a boolean
+         * @return {@code this}.
+         */
+        public B notBooleanBehavior(final ConvertableMappingErrorBehaviour notBooleanBehavior) {
+            this.notBooleanBehavior = notBooleanBehavior;
+            return self();
+        }
+
+        @Override
+        public String toString() {
+            return "PropertyToBoolColumnMapping.PropertyToBoolColumnMappingBuilder(super=" + super.toString()
+                    + ", notBooleanBehavior=" + this.notBooleanBehavior + ")";
+        }
+    }
+
+    private static final class PropertyToBoolColumnMappingBuilderImpl extends
+            PropertyToBoolColumnMapping.PropertyToBoolColumnMappingBuilder<PropertyToBoolColumnMapping, PropertyToBoolColumnMapping.PropertyToBoolColumnMappingBuilderImpl> {
```

Comment:
Would it be nicer to rename the inner classes to just `Builder`?
So it could read  `PropertyToBoolColumnMapping.Builder<PropertyToBoolColumnMapping, PropertyToBoolColumnMapping.Builder>`?
The last class optionally could also be named `PropertyToBoolColumnMapping.BuilderImpl` if this is required.

Reply:
Good point. Done.